compiler/colormsg.nim

```diff
@@ -0,0 +1,27 @@
+import terminal, strutils, strformat, options
```

Comment:
Add a comment about what this module does 
`# This module adds colors to the output of the compiler errors using terminal and strformat.` 
or something like that.

Reply:
even doc comment now that https://nim-lang.github.io/Nim/compiler/theindex.html is a thing